thrift/src/main/thrift/atoms/chart.thrift

```diff
@@ -0,0 +1,58 @@
+namespace * contentatom.chart
+namespace java com.gu.contentatom.thrift.atom.chart
+
+enum ChartType {
+    BAR = 0
```

Comment:
only focus on bar graphs for now

its proving a little tricky to model all the chart types, lets focus on one and prove the end to end process whilst we think about the other chart types

Reply:
Is it because the model will change depending on the type, or just our earlier discussion? just asking as if the former then it's probably best to just rename `ChartAtom` as `BarChartAtom` and drop `ChartType` for now (this way we can still create a general atom in the future). but otherwise just ignore 😅

Reply:
Its more that we don't understand when the other chart types are used, so rather than pollute the chart type enum with potentially unused chart types, lets just work on one that we know is used.